templates/CONFIG_TEMPLATE.INI

```diff
@@ -360,13 +357,7 @@
 
 #  sets the space token for storing temporary files (e.g. oversized input
 #  sandbox)
-#DefaultSRMToken = 
-
-#  enables/disables the support of the EDG middleware
-#EDG_ENABLE = True
-
-#  sets the LCG-UI environment setup script for the EDG middleware
-#EDG_SETUP = /afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh
+#DefaultSRMToken =
```

Comment:
I would be in favour of removing them in another PR. There's a bunch of related scripts in `release/tools` which could also just be burned now. It would probably be quite cathartic to whoever wants to give that a go.